src/repr/src/proto/global_id.rs

```diff
@@ -0,0 +1,40 @@
+// Copyright Materialize, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use crate::proto::TryFromProtoError;
+use crate::GlobalId;
+
+include!(concat!(env!("OUT_DIR"), "/global_id.rs"));
+
+impl From<&GlobalId> for ProtoGlobalId {
```

Comment:
Doesn't this already exist? https://dev.materialize.com/api/rust-private/mz_expr/proto/id/struct.ProtoGlobalId.html

Reply:
https://github.com/MaterializeInc/materialize/pull/11627/files#diff-ae96a1abfa287351061df1b13da8b332caba6157b5bf30c940f71df9fcf1cdafL21

Reply:
Ah, didn't see that.